tests/Feature/BookTest.php

```diff
@@ -266,4 +266,86 @@ public function test_book_claim_db_success()
             'claimed' => 1
         ]);
     }
-}
+
+    public function test_book_return_invalid_email()
+    {       
+        $response = $this->putJson('api/books/return/1', ['email' => 'test']);
+        $response->assertStatus(422);
+        $response->assertInvalid('email');
+    }
+
+    public function test_book_not_found_return()
+    {
+        $response = $this->putJson('/api/books/return/564756445323254', [
+            'email' => 'test@test.com'
+        ]);
+        $response->assertStatus(404);
+        $response->assertJson(function (AssertableJson $json) {
+            $json->has('message')
+                ->where(
+                    'message',
+                    'Book 564756445323254 was not found'
+                );
+        });
+    }
+
+    public function test_book_not_currently_claimed()
+    {
+        $book = Book::factory(['claimed_by_name' => null])->create();
+        
+        $response = $this->putJson("api/books/return/$book->id", [
+            'email' => 'test@test.com'
+        ]);
+
+        $response->assertStatus(400); 
+        $response->assertJson(function (AssertableJson $json) use($book) {
+            $json->has('message')
+                ->where(
+                    'message',
+                    "Book $book->id is not currently claimed"
+                );
+        });
+    }
+
+    public function test_book_incorrect__claimed_email()
+    {
+        $book = Book::factory()->create();
```

Comment:
Override the factory's claimed_by_email to explicitly not be wrong@wrong.com.

Reply:
This feels unnecessary, but I take your point